src/components/TweetAuthor.js

```diff
@@ -1,22 +1,27 @@
 import React, { PropTypes } from 'react';
-import './TweetAuthor.css';
+import './style/TweetAuthor.css';
 
 const propTypes = {
-  user: PropTypes.object.isRequired
+  user: PropTypes.object.isRequired,
+  showModal: PropTypes.func.isRequired
```

Comment:
This is also a convention for react components and callbacks being passed in. TweetAuthor what we call a dumb component, it's just passed in props for a user and renders the it. It's passed a click handler callback, as far as it's aware that's all that's going on.